Readme-brigade.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+
+# install docker:
+
+apt-get update
+apt-get install -qy docker.io
+
+# configure kubernetes apt repo:
+
+apt-get install -y apt-transport-https
+curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
+echo "deb http://apt.kubernetes.io/ kubernetes-xenial main" > /etc/apt/sources.list.d/kubernetes.list
+apt-get update
+
+# install kubernetes:
+
+apt-get install -y kubelet kubeadm kubernetes-cni
+
+# disable swap if exists (check: cat /proc/swaps)
+
+swapoff /dev/sdX
+
+# initiate kubeadm (replace --apiserver-advertise-address with the IP of your host):
+
+kubeadm init --pod-network-cidr=10.244.0.0/16 --apiserver-advertise-address=$(/sbin/ifconfig eth0 | grep 'inet addr:' | cut -d: -f2)
+
+# configure unprivileged user:
+
+useradd packet -G sudo -m -s /bin/bash
+passwd packet
+sudo su packet
+
+# configure environment variables:
+
+sudo cp /etc/kubernetes/admin.conf $HOME/
+sudo chown $(id -u):$(id -g) $HOME/admin.conf
+export KUBECONFIG=$HOME/admin.conf
+echo "export KUBECONFIG=$HOME/admin.conf" | tee -a ~/.bashrc
+
+# configure pod network:
+
+kubectl apply -f https://raw.githubusercontent.com/coreos/flannel/master/Documentation/kube-flannel.yml
+kubectl apply -f https://raw.githubusercontent.com/coreos/flannel/master/Documentation/k8s-manifests/kube-flannel-rbac.yml
+
+# taint master (so that containers can run on master):
+
+kubectl taint nodes --all node-role.kubernetes.io/master-
+
+# check is kubernetes is working:
+
+kubectl get all --namespace=kube-system
+
+# install helm:
+    # download desired version from https://github.com/kubernetes/helm/releases:
+
+    wget https://storage.googleapis.com/kubernetes-helm/helm-v2.7.0-linux-amd64.tar.gz
+    tar xf helm-v2.7.0-linux-amd64.tar.gz
+
+    # copy binaries into PATH:
+
+    sudo cp linux-amd64/helm /usr/bin/
+
+    # initiate helm:
+
+    helm init
+
+# create the binaries for brigade
+
+apt-get update
+apt-get -y upgrade
+wget https://storage.googleapis.com/golang/go1.9.2.linux-amd64.tar.gz
+tar -xvf go1.9.2.linux-amd64.tar.gz
+mv go /usr/local
+export GOROOT=/usr/local/go

```

Comment:
you can add this line in .bashrc

so you dont have to set them up after each login